### PR TITLE
[Backend] Introduce Version class

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's introduce Version class.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <<yons.kim@samsung.com>>

### CODE REVIEW

File: src/Tests/Backend/Version.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import {Version} from '../../Backend/Version';
+
+suite('Backend', function() {
+  suite('Version', function() {
+    suite('#contructor()', function() {
+      test('is contructed with values', function() {
+        const major: number = 1;
+        const minor: number = 0;
+        const patch: number = 2;
+        const option: string = '~220424nightly';
```
Comment (jyoungyun) [inline src/Tests/Backend/Version.test.ts:27]: IMHO, there is no need to use `nightly` postfix on version string.
If the version has `~`, it means that it is a nightly version.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by hyunsik-yoon — APPROVED** [review]

File: src/Tests/Backend/Version.test.ts (hunk 1 above)
Comment (hyunsik-yoon) [reply src/Tests/Backend/Version.test.ts:27]: Note: I wonder if there's any standard (?) versioning scheme for nightly version.
For example, Tensorflow has format like `2.10.0.dev20220424`.
It seems that semantic versioning (https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions) accepts only `+` or `-` after `major.min.patch`.
So, I guess that vendors might decide strings after `major.min.patch`, which could be `~`, or `.` (Tensorflow case), or `-` (semantic versioning scheme)

Comment (jyoungyun) [reply src/Tests/Backend/Version.test.ts:27]: @hyunsik-yoon 
We had a lot of discussions about the versioning rule. And with the result of discussion, the current rule was selected.
You can see discussions detail [here](https://github.com/Samsung/ONE/issues/7421).

**Pull request merged**